client/src/components/CreateTodo.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import "../App.css";
+import {commitMutation} from 'react-relay';
+import { graphql } from "babel-plugin-relay/macro";
+import Button from "@button-inc/component-library/Button";
+import Input from "@button-inc/component-library/Input";
+import RelayEnvironment from "../RelayEnvironment";
+
+// need connection when making new one because relay doesn't have existing id
+const CreateTodoMutation = graphql`
+mutation CreateTodoMutation (
+  $connections: [ID!]!
+  $input: CreateTodoInput!) {
+    createTodo(input: $input) {
+      todoEdge @appendEdge (connections: $connections) {
+        node {
+          id
+          task
+        }
+      }
+    }
+  }
+
+`;
+
+function CreateTodo(props) {
+
+  const submit = e => {
+    //@ts-ignore
+    const input = document.getElementById("newTodo").value
```

Comment:
Any suggestions for how to do this better?

Reply:
you could 
- register an `onChange={(event) => doSomething(event.target.value)}` on the input field, to keep an always up-to-date state 
- or better, wrap the whole thing in a `<form onSubmit={handleSubmit}>`, the submit click event should have an array of elements as a target where you can retrieve your data